File: specification/web/resource-manager/Microsoft.DomainRegistration/stable/2022-03-01/examples-js/GetDomainOwnershipIdentifier.js
const { WebSiteManagementClient } = require("@azure/arm-appservice");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Description for Get ownership identifier for domain
 *
 * @summary Description for Get ownership identifier for domain
 * x-ms-original-file: specification/web/resource-manager/Microsoft.DomainRegistration/stable/2022-03-01/examples/GetDomainOwnershipIdentifier.json
 */
async function getDomainOwnershipIdentifier() {
  const subscriptionId = "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345";
  const resourceGroupName = "testrg123";
  const domainName = "example.com";
  const name = "SampleOwnershipId";
  const credential = new DefaultAzureCredential();
  const client = new WebSiteManagementClient(credential, subscriptionId);
  const result = await client.domains.getOwnershipIdentifier(resourceGroupName, domainName, name);
  console.log(result);
}

getDomainOwnershipIdentifier().catch(console.error);

File: specification/web/resource-manager/Microsoft.DomainRegistration/stable/2022-03-01/examples-java/GetDomainOwnershipIdentifier.java
import com.azure.core.util.Context;

/** Samples for Domains GetOwnershipIdentifier. */
public final class Main {
    /*
     * x-ms-original-file: specification/web/resource-manager/Microsoft.DomainRegistration/stable/2022-03-01/examples/GetDomainOwnershipIdentifier.json
     */
    /**
     * Sample code: Get Domain Ownership Identifier.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void getDomainOwnershipIdentifier(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .webApps()
            .manager()
            .serviceClient()
            .getDomains()
            .getOwnershipIdentifierWithResponse("testrg123", "example.com", "SampleOwnershipId", Context.NONE);
    }
}
